stdlib/2and3/ssl.pyi

```diff
@@ -134,7 +134,7 @@ PROTOCOL_SSLv3: int
 PROTOCOL_TLSv1: int
 PROTOCOL_TLSv1_1: int
 PROTOCOL_TLSv1_2: int
-if sys.version_info >= (3, 5):
+if sys.version_info >= (3, 5) or (2, 7, 13) <= sys.version_info < (3, 0):
```

Comment:
```suggestion
if sys.version_info < (3, 0) or sys.version_info >= (3, 5):
```

Looks like we don't indicate the lower bound in all other similar cases and match better the recommendations from https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#stub-versioning

Reply:
Hmm, I see. So it looks like stubs only support 'version_info >= <lower_bound>`, which means we can't do 2.7-only conditions, eg `(2, 7) <= sys.version_info < (3, 0)`?

Reply:
Pushed a commit for that since it actually seems to work. :-)

Reply:
Actually we don't need a version check here at all, since Python 3.0 through 3.4 are not supported in typeshed, meaning `PROTOCOL_TLS` exists in all supported versions of typeshed.